src/vs/workbench/contrib/search/browser/searchResultsView.ts

```diff
@@ -401,10 +401,10 @@ export class SearchAccessibilityProvider implements IListAccessibilityProvider<R
 			const range = match.range();
 			const matchText = match.text().substr(0, range.endColumn + 150);
 			if (replace) {
-				return nls.localize('replacePreviewResultAria', "Replace '{0}' with '{1}' in line '{2}' at column {3}", matchString, match.replaceString, matchText, range.startColumn + 1);
+				return nls.localize('replacePreviewResultAria', "'{0}' at column {1} replace {2} with {3}", matchText, range.startColumn + 1, matchString, match.replaceString);
```

Comment:
this is nitpicky, but should there be a comma or semicolon before replace? Not sure what's correct grammar in this case.

Reply:
For screen readers, we want to keep content as brief as possible. 

The first part is a sentence fragment; "result at column 9". We'd have to include a verb to make it an independent clause in order to justify punctuation imo. We don't want to add any more words for the reason mentioned above.

https://user-images.githubusercontent.com/29464607/232152264-07aae30e-76f9-4953-8d33-24a47b5dd6e5.mov